components/service/glean/src/main/java/mozilla/components/service/glean/storages/TimingDistributionsStorageEngine.kt

```diff
@@ -262,12 +272,12 @@ data class TimingDistributionData(
         return JSONObject(mapOf(
             "category" to category,
             "name" to name,
-            "bucketCount" to bucketCount,
+            "bucket_count" to bucketCount,
             "range" to JSONArray(arrayOf(rangeMin, rangeMax)),
-            "histogramType" to histogramType.ordinal,
+            "histogram_type" to histogramType.toString().toLowerCase(),
```

Comment:
Great catch. Gah, I missed all these things in the review a while back. Shame on me.

Reply:
No worries.  We're "eventually consistent"